### ISSUE TITLE
构建失败。

### ISSUE THREAD

**Issue by unlceli** [JsonChao/Awesome-WanAndroid#78]
Unable to resolve dependency for ':app@debug/compileClasspath': Could not download accessibility-test-framework.jar (com.google.android.apps.common.testing.accessibility.framework:accessibility-test-framework:2.1)
Open File
Show Details

无论是代理，离线，翻墙，都不行，求指教。

**Comment by JsonChao** [comment]
Thank your issue，对于这种问题建议谷歌搜寻关键字android studio Could not download，应该可以解决你的问题~
